src/main/resources/static/_shared.yml

```diff
@@ -3073,6 +3073,9 @@ components:
                 $ref: '#/components/schemas/ApprovedPremisesUserRole'
             apArea:
```

Comment:
Do we need to set deprecated: true here and add a message to use cruManagementArea instead?

Reply:
Great point - i've made this change as apArea doesn't make much sense in this context anymore